stdlib/crypto/src/main/java/org/ballerinalang/stdlib/crypto/nativeimpl/Crc32b.java

```diff
@@ -42,11 +40,7 @@
 @BallerinaFunction(
         orgName = "ballerina", packageName = "crypto",
         functionName = "crc32b", isPublic = true)
-public class Crc32b extends BlockingNativeCallableUnit {
-
-    @Override
-    public void execute(Context context) {
-    }
+public class Crc32b {
```

Comment:
Since the classes only have static functions we can add a private constructor to make sure these classes cannot be initialized.

Reply:
What is the use case that involves initializing these classes?

Reply:
That is a violation of a sonar rule. If a class has only static functions we need to prevent it from being intialized: https://rules.sonarsource.com/java/RSPEC-1118

Reply:
This is not utility class. This is a part of the framework implementation.

Sonar rule says "Utility classes should not have public constructors"

Reply:
It is to prevent a class from being instantiated. I think these classes should not be instantiated.